src/main/java/org/isf/medstockmovtype/model/MovementType.java

```diff
@@ -30,16 +30,15 @@
 import javax.persistence.Transient;
 import javax.validation.constraints.NotNull;
 
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 import org.isf.utils.db.Auditable;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 /**
- * ------------------------------------------
- * MovementType - model for the movement type
- * -----------------------------------------
- * modification history
- * ? - bob - first version
- * 18/01/2015 - Antonio - ported to JPA
+ * ------------------------------------------ MovementType - model for the
```

Comment:
I wasn't going to comment about all the space/reformatting going on but here again a nicely formatted comment block is mashed together making it hard to read.